test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -25,8 +25,35 @@
 
 namespace Microsoft.ML.Scenarios
 {
+
+    internal sealed class TensorFlowScenariosTestsFixture : IDisposable
+    {
+        public TensorFlowScenariosTestsFixture()
+        {
+            string assetsRelativePath = @"assets";
+            string assetsPath = GetAbsolutePath(assetsRelativePath);
+            string workspacePath = Path.Combine(assetsPath, "cached");
```

Comment:
lets make this a function that reuse it here and below in the tests. Any time a piece of code is reused (with very few exceptions) you want to make it a function and use it. It makes the code more maintainable and readable.